### ISSUE TITLE
Can't requestBaseUrl be undefined?

### ISSUE THREAD

**Issue by x-yuri** [Lakitna/cypress-commands#26]
# Describe the bug
`cy.request()` fails with:

> TypeError: Cannot read property 'length' of undefined

when no `requestBaseUrl` present in `cypress.json`.

# To Reproduce
`cypress/integration/1.spec.js`:

```js
describe('cy.request', function() {
    it('succeeds', () => {
        cy.request('GET', '/some/relative/url');
    });
});

```

# Expected behavior
It succeeds.

# Versions and such:
 - OS: Arch Linux
 - Cypress version: 3.4.0
 - Cypress browser: Electron 61
 - Cypress-commands version: 0.2.0

# Additional context
The culprit line is [this one][1].

[1]: https://github.com/Lakitna/cypress-commands/blob/1b2127367343974151e0490eaca3f5052999ce12/src/request.js#L81

**Comment by Lakitna** [comment]
Oh wow, that's a weird thing not to catch! I'll add the fix to 0.3.0

**Comment by Lakitna** [comment]
Will be part of 0.3.0
https://github.com/Lakitna/cypress-commands/milestone/2
